fsm/cli
# Delete embedded traverser, allow user to pass in

Fixes #1 
Fixes #2 

- Remove embedded traverser.  Users will now need to pass in a store
  - This gives a more consistent experience as compared to other targets
  - Allows users to test stores easily
- Only thing exportable is now `Start`, which is the only thing users should be interfacing with
- Add some inline comments / minor code health refactors
- Add in sleeps after most emitables, to make the conversation feel a bit more natural.
